geps/gep-1619/index.md

```diff
@@ -317,71 +346,55 @@ In this section, we will explore the questions and design elements associated wi
 ### GO
 
 ```go
-// SessionPersistencePolicy provides a way to define session persistence rules
-// for a service or route.
+// LoadBalancerPolicy provides a way to define load balancing rules
+// for a service.
 //
-// Support: Core
-type SessionPersistencePolicy struct {
+// Support: Extended
+type LoadBalancerPolicy struct {
     metav1.TypeMeta   `json:",inline"`
     metav1.ObjectMeta `json:"metadata,omitempty"`
 
-    // Spec defines the desired state of SessionPersistencePolicy.
-    Spec SessionPersistencePolicySpec `json:"spec"`
+    // Spec defines the desired state of LoadBalancerPolicy.
+    Spec LoadBalancerPolicySpec `json:"spec"`
 
-    // Status defines the current state of SessionPersistencePolicy.
-    Status SessionPersistencePolicyStatus `json:"status,omitempty"`
+    // Status defines the current state of LoadBalancerPolicy.
+    Status LoadBalancerPolicyStatus `json:"status,omitempty"`
 }
 
-// SessionPersistencePolicySpec defines the desired state of
-// SessionPersistencePolicy.
+// LoadBalancerPolicySpec defines the desired state of
+// LoadBalancerPolicy.
 // Note: there is no Override or Default policy configuration.
-type SessionPersistencePolicySpec struct {
+type LoadBalancerPolicySpec struct {
     // TargetRef identifies an API object to apply policy to.
-    // The TargetRef may be a Service, HTTPRoute, GRPCRoute,
-    // or a HTTPRouteRule or GRPCRouteRule section.
-    // At least one of these targets must be supported for
-    // core-level compliance.
-    //
+    // Currently, Services are the only valid API target references.
```

Comment:
Should it say 'backends' (i.e. anything that is allowed to be a backendRef - like clusterset or other vendor specific backends ) ?

Reply:
Yea, at the current moment, I don't see why we should prevent implementations from using a custom backend or `ServiceImport` for multi-cluster services. I'll include it.